miqa/core/rest/other_endpoints.py

```diff
@@ -27,7 +27,7 @@ class MIQAConfigView(APIView):
     def get(self, request):
         return Response(
             {
-                'artifact_options': [key for key in default_identified_artifacts()],
+                'artifact_options': list(default_identified_artifacts()),
                 'auto_artifact_threshold': 0.4,
```

Comment:
Should this threshold also be configurable on a per-project level, or are we OK with it remaining global?

Reply:
This makes sense to me. Maybe a separate issue can be added? I'd like to avoid expanding the scope atm.

Reply:
Definitely, let's do it on a follow-up.